src/main/java/us/abstracta/wiresham/Flow.java

```diff
@@ -144,9 +169,65 @@ public static Flow fromPcap(File file, String serverAddress, String filter) thro
   }
 
   public static Flow fromYml(File ymlFile) throws FileNotFoundException {
-    List<PacketStep> packets = new Yaml(buildYamlConstructor())
-        .load(new FileInputStream(ymlFile));
-    return new Flow(packets);
+    List<Flow> flows = new ArrayList<>();
+    new Yaml(buildYamlConstructor())
```

Comment:
An alternative can be to use streams with something like `StreamSupport.stream(iterable, false)`

Reply:
And use flatmap for parseFlowsFromYml?